tagger/model/QKerasModel.py

```diff
@@ -154,8 +154,10 @@ def load(self, out_dir: str = "None"):
 
         # Additional custom objects for attention layers
         custom_objects_ = {
+            "NodeEdgeProjection": NodeEdgeProjection,
```

Comment:
This `NodeEdgeProjection` isn't imported, and I suppose it can't be as-is without creating a circular import. `NodeEdgeProjection`'s class definition might have to move to `common.py`

Reply:
Thought I solved this by redefining this load method for the interaction net, leads to code replication but means we don't have to bring in the NodeEdgeProjection in common